test/units/TEST-26-SYSTEMCTL.sh

```diff
@@ -343,6 +343,12 @@ systemctl cat "$UNIT_NAME"
 systemctl help "$UNIT_NAME"
 systemctl service-watchdogs
 systemctl service-watchdogs "$(systemctl service-watchdogs)"
+# Ensure that the enablement symlinks can still be removed after the user is gone, to avoid having leftovers
+systemctl enable "$UNIT_NAME"
+systemctl stop "$UNIT_NAME"
+rm -f "/usr/lib/systemd/system/$UNIT_NAME"
+systemctl daemon-reload
+systemctl disable "$UNIT_NAME"
```

Comment:
Hmm, I think the general concept is OK, but we should still propagate the "not-found" error as final result.

Reply:
That happens when nothing is found - but if something is disabled (symlink is removed), then it is successful, which sounds like what we want? IE, the relevant thing is that the disable operation succeeded and did its job